Makefile

```diff
@@ -56,6 +56,9 @@ dev: 						        ## Run the development app (and run extra migrations first)
 test:                               ## Execute tests
 	$(dc) run --rm test pytest /tests $(ARGS)
 
+parser:
```

Comment:
ik zou parser hernoemen naar `telcameras_v2_parser`, tenzij dit de enige blijft. Maar gezien de setup met de abstracte class lijkt het daar niet op

Reply:
Dit geldt zowel hier als in docker-compose